ui.go

```diff
@@ -296,10 +296,12 @@ func (win *Win) printd(dir *Dir, marks map[string]bool) {
 					tmp += fmt.Sprintf("%s ", f.ModTime().Format("Jan _2 15:04"))
 				}
 			}
-			s = runeSliceWidthRange(s, 0, win.w-3-len(tmp))
-			s = append(s, ' ')
-			for _, r := range tmp {
-				s = append(s, r)
+			if win.w-len(tmp)-10 > 3 {
```

Comment:
@IvanMenshykov Previous heuristic was to hide the info if it would exceed half the column width. We can discuss to change that heuristic of course but I can't make sense out of this currently. Where does `10` come from?